packages/@guardian/source-foundations/src/typography/fs.ts

```diff
@@ -27,31 +67,27 @@ function getFontStyle(
 export const fs: Fs =
 	(category) =>
 	(level, { lineHeight, fontWeight, fontStyle, unit }) => {
-		const fontFamilyValue = fontMapping[category];
+		const fontFamilyValue = fonts[category];
 		const fontSizeValue: `${number}rem` | number =
 			unit === 'px'
-				? Number(fontSizeMapping[category][level])
-				: // @ts-expect-error -- the types actually overlap, see https://gist.github.com/mxdvl/5e31fd5b13670b6a41ddac6c65efeee4
-				  `${Number(remFontSizeMapping[category][level])}rem`;
+				? Number(pxTextSizes[category][level])
+				: `${Number(remTextSizes[category][level])}rem`;
 		const lineHeightValue: `${number}px` | number =
 			unit === 'px'
 				? // line-height is defined as a unitless value, so we multiply
 				  // by the element's font-size in px to get the px value
 				  `${
-						lineHeightMapping[lineHeight] *
```

Comment:
We are able to completely remove `lineHeightMapping` and `fontSizeMapping` by referencing the definition directly from data.ts